extensions/blocks/dialogue/components/time-stamp-control.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Dropdown,
+	Button,
+	__experimentalNumberControl as NumberControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+function validateValue( val, max ) {
+	return Math.max( 0, Math.min( val, max ) );
+}
+
+const timeStampMap = [ 'hour', 'min', 'sec' ];
```

Comment:
I'm nit-picking, but I believe "timestamp" is in one word, so we could renamed `time-stamp-control.js` into `timestamp-control.js` and switch our camel case usage to `timestampMap`. What do you think?

Reply:
agree